src/containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/AMMAccountHeader.tsx

```diff
@@ -39,11 +40,17 @@ export const AMMAccountHeader = (props: { data: AmmDataType }) => {
           </div>
           <div className="values">
             <div className="title">{t('token_balance')}</div>
-            <div className="value">{localizedBalance1}</div>
+            {/* <div className="value">{localizedBalance1}</div> */}
```

Comment:
These commented lines should be removed and the variables should be deleted as well

Reply:
@mvadari i kept those commented lines for your reference

Reply:
You don't need to do that - it shows up in the diff